runbot_send_email/tests/test_runbot_send_email.py

```diff
@@ -0,0 +1,102 @@
+# coding: utf-8
+# © 2015 Vauxoo
+#   Coded by: lescobar@vauxoo.com
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+import os
+import threading
+
+from openerp import exceptions
+from openerp.tests.common import TransactionCase
+
+_logger = logging.getLogger(__name__)
+
+
+class TestRunbotSendEmail(TransactionCase):
+    """
+    This create a runbot send email test
+    """
+
+    def setUp(self):
+        """
+        This add required environment variable for test
+        """
+        super(TestRunbotSendEmail, self).setUp()
+        repo_name = 'https://github.com/vauxoo-dev/'
+        repo_name += 'runbot_branch_remote_name_grp_feature2.git'
+        branch_name = 'refs/heads/fast-travis'
+        self.repo_obj = self.env['runbot.repo'].create({
+            'name': repo_name,
+            'modules_auto': 'repo',
+            'mode': 'poll',
+            'nginx': True,
+        })
+        self.branch_obj = self.env['runbot.branch'].create({
+            'repo_id': self.repo_obj.id,
+            'name': branch_name,
+            'branch_name': 'fast-travis',
+            'sticky': False,
+            'coverage': False,
+        })
+        recipient = 'testcommitter@runbot.vauxoo.com'
+        self.build_obj = self.env['runbot.build'].create({
+            'branch_id': self.branch_obj.id,
+            'name': 'fcb98eb195fc62fa49873f8f101f1738e38ea7c0',
+            'author': 'Test Author',
+            'author_email': 'author@runbot.vauxoo.com',
+            'committer': 'Test Committer',
+            'committer_email': recipient,
+            'subject': '[TEST] Test message commit',
+            'date': '2016-03-08 00:00:00',
+       })
+        domain = [ ('repo_id','=',self.repo_obj.id), ]
+        self.build = self.build_obj.search(domain)
+        if os.environ.get('EMAIL_PASSWORD', False):
+            setattr(threading.currentThread(), 'testing', False)
+            self.build.pool._init = False
+            self.mail_server = self.env['ir.mail_server'].create({
```

Comment:
Check the example: https://github.com/Vauxoo/yoytec/blob/8.0/mail_notification_collection/demo/ir_mail_server_demo.xml